tests/themes-parsing.spec.js

```diff
@@ -9,6 +9,8 @@ var plist = require('plist'),
 	libxml = require('libxmljs'),
 	colorParser = require('color-parser'),
 	fs = require('fs'),
+	et = require('elementtree'),
+	assert = require('assert'),
```

Comment:
we use `chai + expect` for our unit test, please replace assert tests with `expect`

Reply:
done! Sorry, I hadn't looked at the runner